ecs-cli/modules/cli/local/up_app.go

```diff
@@ -93,14 +97,38 @@ func createNewComposeProject(c *cli.Context) (string, error) {
 	return filepath.Abs(project.LocalOutFileName())
 }
 
-func runContainers(composePath string) {
+func composeOverridePaths(basePath string, additionalRelPaths []string) ([]string, error) {
+	defaultPath := basePath[:len(basePath)-len(filepath.Ext(basePath))] + ".override.yml"
+	paths := []string{defaultPath}
+	if len(additionalRelPaths) > 0 {
+		for _, relPath := range additionalRelPaths {
+			p, err := filepath.Abs(relPath)
+			if err != nil {
+				return nil, errors.Wrapf(err, "cannot get absolute path of %s", relPath)
+			}
+			paths = append(paths, p)
+		}
+	}
+
+	// Prune paths that don't exist
+	var overridePaths []string
+	for _, p := range paths {
+		if _, err := os.Stat(p); err == nil {
+			overridePaths = append(overridePaths, p)
+		} else {
+			logrus.Warnf("Skipping Compose file %s due to:\n%v", filepath.Base(p), err)
```

Comment:
There also can be a caveat for duplication since the path is not guaranteed to be unique.

Reply:
I just tested this, I think it's fine if a user specifies the same file multiple times. There is no error that happens